lib/src/core/verifier/policy/opa/mod.rs

```diff
@@ -102,34 +102,27 @@ svn_old_allow {
     fn test_evaluate() {
         let policy = dummy_policy().unwrap();
 
-        let res = evaluate(
-            policy.clone(), 
-            dummy_reference(3), 
-            dummy_input(5, 5)
-        );
+        let res = evaluate(policy.clone(), dummy_reference(3), dummy_input(5, 5));
         assert!(res.is_ok(), "OPA execution() should success");
         let v: Value = serde_json::from_str(&res.unwrap()).unwrap();
         assert!(v["allow"] == true, "allow should true");
-        assert!(v["product_id_allow"] == true, "product_id_allow should true");
+        assert!(
+            v["product_id_allow"] == true,
+            "product_id_allow should true"
```

Comment:
I made a mistake because these changes should not be included in this PR. I will revert it. Or it will conflict with the "Add basic compile testing" PR.